pages.zh/linux/flameshot.md

```diff
@@ -0,0 +1,23 @@
+# flameshot
+
+带有gui界面的Screenshot工具，支持基本的图像编辑，例如文本，形状，颜色和imgur。更多信息: <https://flameshot.js.org>
```

Comment:
Missing syntax here
```suggestion
> 带有gui界面的Screenshot工具，支持基本的图像编辑，例如文本，形状，颜色和imgur。更多信息: <https://flameshot.js.org>.
```

The `> ` indicates a blockquote. The full-stop at the end is required by the linter (unfortunately the linter was implemented before multiple languages were supported by tldr-pages).